### PR TITLE
Split-up of amp-bind-integrations.html for dynamic-tags

### PR DESCRIPTION
**Pull request by sreenidhipm**
Fixes amp-bind: Speed up integration tests #8197
* Part 2: Split up the amp-bind-integrations.html test fixture into multiple files under a single new directory in test/fixtures/
* Split up for dynamic-tags

### CODE REVIEW

**Comment by sreenidhipm** [comment]
@kmh287 Separate PR for dynamic-tags

File: test/fixtures/amp-bind-integrations/dynamic-tags.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-bind integration test</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
```
Comment (kmh287) [inline test/fixtures/amp-bind-integrations/dynamic-tags.html]: No need for the original and new CSS

Comment (sreenidhipm) [inline test/fixtures/amp-bind-integrations/dynamic-tags.html]: Done!
Added commits for having a separate test file using this fixture as well.

File: extensions/amp-bind/0.1/test/test-bind-integration-dynamictags.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        waitForEvent('amp:bind:setState'));
+  }
+
+  /** @return {!Promise} */
+  function waitForAllMutations() {
+    return bindForDoc(ampdoc).then(unusedBind =>
+        waitForEvent('amp:bind:mutated'));
+  }
+
+  describe('detecting bindings under dynamic tags', () => {
```
Comment (kmh287) [inline extensions/amp-bind/0.1/test/test-bind-integration-dynamictags.js]: Similar to https://github.com/ampproject/amphtml/pull/8435#discussion_r109207830, let's remove this describe block and combine the text with the top-level description i.e.

`amp-bind dynamic tag integration`

File: test/fixtures/amp-bind-integrations/dynamic-tags.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+</head>
+<body>
+  <p>DYNAMIC TAGS</p>
+  <!-- For generic dynamic tag tests only. Template tag used as an example. -->
+  <div>
+    <template id="dynamicTemplate"></template>
+    <!-- Siblings added in test -->
+  </div>
+</body>
+</html>
```
Comment (kmh287) [inline test/fixtures/amp-bind-integrations/dynamic-tags.html]: nit: newline

File: test/fixtures/amp-bind-integrations/dynamic-tags.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  <meta charset="utf-8">
+  <title>amp-bind integration test</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script src="/dist/amp.js"></script>
+  <script custom-element="amp-bind" src="/dist/v0/amp-bind-0.1.max.js"></script>
+</head>
+<body>
+  <p>DYNAMIC TAGS</p>
```
Comment (kmh287) [inline test/fixtures/amp-bind-integrations/dynamic-tags.html]: I had to change the HTML for the dynamic tags test due to a bug in bind. Could you please pull in master and update this file accordingly?

**Comment by kmh287** [comment]
Hey @sreenidhipm , are you still working on this? Let me know if you have any questions.

**Pull request closed without merging**